### ISSUE TITLE
Add Cognito template to Auth module

### ISSUE THREAD

**Issue by arantespp** [ttoss/modules#35]
Auth module will also provide the CloudFormation template to the user. This functionality is necessary because the auth flow is significantly correlated with the cloud resource created.

Tasks:

- [ ] Transform [this template](https://github.com/TriangulosTecnologia/siflor/blob/main/packages/cloud/auth/cloudformation.yml) in a JavaScript module and export it.
- [ ] Create an example of the template on the ttoss AWS account.

**Comment by arthurlolico** [comment]
I have a question about the second task.

Create an example of the template would be make an instance of the template by import it from cloudformation.ts and put dat on it?

**Comment by arantespp** [comment]
The example is about creating a package, importing the template, and deploying the resources to AWS.

**Comment by arthurlolico** [comment]
I get it. By "creating a package", would be something like this?

https://docs.npmjs.com/creating-node-js-modules

**Comment by arantespp** [comment]
Exactly. You should also create the structure specified on carlin documentation.

**Comment by arthurlolico** [comment]
I have another question. What kind of package is this? Unscoped, scoped, public, private?

**Comment by arantespp** [comment]
You could see the other packages inside `examples` folder. They're private and scoped.
